build-tools/scripts/MonoAndroid.targets

```diff
@@ -0,0 +1,48 @@
+<!--
+    Import this file at the bottom, when you need to build a
+    MonoAndroid assembly in the local build tree.
+-->
+<Project>
+  <PropertyGroup>
+    <LanguageTargets Condition="Exists('$(OutputPath)\..\..\..\xbuild\Xamarin\Android\Xamarin.Android.CSharp.targets')">$(OutputPath)\..\..\..\xbuild\Xamarin\Android\Xamarin.Android.CSharp.targets</LanguageTargets>
+  </PropertyGroup>
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
+
+  <!-- References -->
+  <ItemGroup>
+    <Reference Include="mscorlib">
+      <HintPath>$(OutputPath)..\v1.0\mscorlib.dll</HintPath>
+      <Private>false</Private>
+    </Reference>
+    <Reference Include="System">
+      <HintPath>$(OutputPath)..\v1.0\System.dll</HintPath>
+      <Private>false</Private>
```

Comment:
What about the `<Reference />`s here. Wouldn't it make more sense to have them in the project files? `System.EnterpriseServices.csproj` doesn't need these it looks.

Do we need the `HintPath`?

Reply:
`<Csc/>` should drop the `@(Reference)` that aren't used, I was mainly trying to have as much common between the three projects.

The `HintPath` is needed here because these three projects are built with regular MSBuild (not xabuild) and they use `$(NoStdLib)`.